.github/ISSUE_TEMPLATE/scraper_bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Scraper bug report
+about: Report a scraper that is not working correctly
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Thanks for filing a bug report with us!  To help get the issue fixed, please fill in the information below.
+
```

Comment:
Can we ask for Python version and OS? We recently had an issue that was potentially OS specific and it seems that all 3 of us maintainers run some sort of *nix.